docs/BUILD.md

```diff
@@ -205,11 +205,13 @@ To build `installer.raw`:
     3. Populates each partition with its appropriate contents:
         * `efi`: contents of `/EFI/BOOT/` from `/parts/rootfs.img`
         * `imga`: contents of `/parts/rootfs.img`
-        * `conf_win`: contents of `/parts/config.img`. `conf_win` is different from `conf` only in the partition type. It is unclear why this matters for the installer.
+        * `conf_win`: contents of `/parts/config.img`. `conf_win` is different from `conf` only in the partition type. This is required so that this partition can be mounted on MacOS and Windows machines for users to add/change configuration on the fly. 
```

Comment:
> only in the partition type

Ah, I see, from [pkg/mkimage-raw-efi/make-raw](https://github.com/zededa/zenbuild/blob/master/pkg/mkimage-raw-efi/make-raw#L151-L157):

```sh
function do_conf {
    do_vfat $1 13307e62-cd9c-4920-8f9b-91b45828b798
}

function do_conf_win {
    do_vfat $1 EBD0A0A2-B9E5-4433-87C0-68B6B72699C7
}
```

* `conf` has partition GUID `13307e62-cd9c-4920-8f9b-91b45828b798` which is what type? It doesn't appear in the list in [wikipedia](https://en.wikipedia.org/wiki/GUID_Partition_Table) or most other sources I hunted down.
* `conf_win` has partition GUID `EBD0A0A2-B9E5-4433-87C0-68B6B72699C7` which is Windows basic data partition.

> This is required so that this partition can be mounted on MacOS and Windows machines for users to add/change configuration on the fly

Is the assumption that `installer` would be run on a local laptop installing to a connected sd card or similar? But the installer is a bootable "thing", wouldn't need that. So is it just to update the configuration information? Could we have a Win/Mac utility that flashes it correctly? Or is the desire to be able to inspect it? 

Conversely, is there not a single partition type mountable across everyone?